CONTRIBUTING.md

```diff
@@ -1,53 +1,307 @@
+Since we do not edit specification documents after publication, even the smallest change has to be in a new release.
+
+Changes in patch releases meet the following criteria:
+
+* Editorial changes such as spelling or formatting fixes, including link updates.
+* Clarifications or additions that do not change the meaning of the specification.
+
+Patch releases are created as often as there are changes to the specification worth releasing.
+
+### Release Process
+
+A release requires a vote on the specification at a particular version and the associated release notes by TSC members within the voting period.
+Major or minor release voting periods will be announced in the Slack channel and noted on the calendar at least 6 days in advance.
+During this time, TSC members who have not yet voted must note their approval by leaving a comment on the GitHub pull request proposing the release; release notes should be included with the description.
+TSC members are responsible for coordinating the information about the release to the outreach team as appropriate.
+
+* Patch-level releases require majority approval by TSC members. (Max voting period 3 days)
+
+* Minor: requires approval by 66% of TSC members. (Max voting period 7 days)
+
+* Major: requires approval by 66% of TSC members. (Max voting period 14 days)
+
+During the voting period, further changes should not be made to the specification being considered.
+
+Once the threshold of approvals is met, the release can be performed by any TSC member.
+
+## Propose a Specification Change
+
+As an organisation, we're open to changes, and these can be proposed by anyone.
+The specification is very widely adopted, and there is an appropriately high bar for wide appeal and due scrutiny as a result.
+We do not accept changes lightly (but we will consider any that we can).
+
+Small changes are welcome as pull requests.
+
+Bigger changes require a more formal process.
+
+1. Start a [discussion](https://github.com/OAI/OpenAPI-Specification/discussions) of type "Enhancements".
+   The discussion entry must include some use cases, your proposed solution and the alternatives you have considered.
+   If there is engagement and support for the proposal over time, then it can be considered as a candidate to move to the next stage.
+
+2. It really helps to see the proposed change in action.
+   Start using it as a `x-*` extension if that's appropriate, or try to bring other evidence of your proposed solution being adopted.
+
+3. If you are adding support for something from another specification (such as OAuth), please point to implementations of that
+   specification so that we can understand how, and to what degree, it is being used.
+
+4. If the suggested change has good support, you will be asked to create a formal proposal.
+   Use the [template in the proposals directory](https://github.com/OAI/OpenAPI-Specification/tree/main/proposals), copy it to a new file, and complete it.
```

Comment:
```suggestion
   Use the [template in the proposals directory](https://github.com/OAI/OpenAPI-Specification/tree/main/proposals/2019-01-01-Proposal-Template.md), copy it to a new file, and complete it.
```

Reply:
I'll skip this, people should look in the directory and find a template. The current one has some problems and is stupidly named, so just directing people to the directory means this file won't be outdated if I ever fix that problem!